godot-core/src/builtin/variant/impls.rs

```diff
@@ -171,6 +171,53 @@ mod impls {
     impl_ffi_variant!(ref PackedColorArray, packed_color_array_to_variant, packed_color_array_from_variant);
     impl_ffi_variant!(ref Signal, signal_to_variant, signal_from_variant);
     impl_ffi_variant!(ref Callable, callable_to_variant, callable_from_variant);
+
+    #[cfg(since_api = "4.2")]
```

Comment:
Could you add a module `impls`, then you need the cfg only once.

Inside it, you can then also
```rs
use crate::task::impl_maybe_send;
```